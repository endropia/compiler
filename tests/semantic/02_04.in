var
	i: char;
begin
	for i := 0 to 3 do begin end;
end.