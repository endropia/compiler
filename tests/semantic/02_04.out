(4, 6) Ordinary expected in for